Specs/Scene/ModelSpec.js

```diff
@@ -88,7 +88,9 @@ defineSuite([
     var boxSpotLightUrl = './Data/Models/MaterialsCommon/BoxSpotLight.gltf';
     var boxTransparentUrl = './Data/Models/MaterialsCommon/BoxTransparent.gltf';
     var boxColorUrl = './Data/Models/Box-Color/Box-Color.gltf';
+    var boxUint32Indices = './Data/Models/Box-Uint32Indices/Box-Uint32Indices.gltf';
     var boxQuantizedUrl = './Data/Models/WEB3DQuantizedAttributes/Box-Quantized.gltf';
+    var boxNormalizedQuantizedUrl = './Data/Models/WEB3DQuantizedAttributes/Box-Normalized-Quantized.gltf';
```

Comment:
I actually just removed this from #4247. I'll merge up the chain when that gets merged.